config/environments/development.rb

```diff
@@ -33,17 +44,23 @@
   config.eager_load = false
 
   require 'rack-webauth/test'
-  config.middleware.use(
-    Rack::Webauth::Test,
-    user: Settings.WEBAUTH.USER,
-    mail: Settings.WEBAUTH.MAIL,
-    ldapprivgroup: Settings.WEBAUTH.LDAPPRIVGROUP,
-    suaffiliation: Settings.WEBAUTH.SUAFFILIATION,
-    displayname: Settings.WEBAUTH.DISPLAYNAME,
-    ldapauthrule: Settings.WEBAUTH.LDAPAUTHRULE
-  )
+  if Settings.WEBAUTH
```

Comment:
thanks!

any other stuff left to clean up with this one before merging?  i think this is the last of the issues raised so far...